### ISSUE TITLE
The "set" methods are not working in Mpay24Order.php

### ISSUE THREAD

**Issue by mdaskalov** [mpay24/mpay24-php#95]
It is not possible to set attributes using the set...() function anymore. An exception `At least one Argument must be provided...` is always thrown.

Example:
`$mdxi->Order->BillingAddr->setMode("ReadOnly");`
Throws: 
`Fatal error: Uncaught InvalidArgumentException: At least one Argument must be provided for node 'setMode'! in /var/www/html/mpay24-php/src/Mpay24Order.php:114 Stack trace: #0 /var/www/html/mpay24-php-tests/min_paypage.php(23): Mpay24\Mpay24Order->__call('setMode', Array) #1 {main} thrown in /var/www/html/mpay24-php/src/Mpay24Order.php on line 114`

There should be a return after line 79

**Comment by mdaskalov** [comment]
It seems that it was wrong since the beginning as nothing was returned when the keyword 'set' was met (in the first if). The following should work:
`$mdxi->Order->setAttribute1("value1")->setAttribute2("value2");`

We should create some unit tests for this...

For example this:
```
$mdxi   = new Mpay24Order();

$mdxi->Order->Tid = "123 TID";
$mdxi->Order->TemplateSet->setCSSName("MODERN")->setLanguage("DE");
$mdxi->Order->ShoppingCart->Description = "Test Payment";
$mdxi->Order->ShoppingCart->Item(1)->Number = null;
$mdxi->Order->ShoppingCart->Item(1)->ProductNr = 'abc';
$mdxi->Order->ShoppingCart->Item(1)->Description = 'Test Product 1';
$mdxi->Order->ShoppingCart->Item(1)->Quantity = 1;
$mdxi->Order->ShoppingCart->Item(1)->Price = '0.25';
$mdxi->Order->ShoppingCart->Item(2)->Number = null;
$mdxi->Order->ShoppingCart->Item(2)->ProductNr = 'abc';
$mdxi->Order->ShoppingCart->Item(2)->Description = 'Test Product 1';
$mdxi->Order->ShoppingCart->Item(2)->Quantity = 1;
$mdxi->Order->ShoppingCart->Item(2)->Price = '0.75';

$mdxi->Order->Price = "1.00";

$mdxi->Order->Customer = 'Max Muster';
$mdxi->Order->Customer->setId("10002")->setUseProfile("true");

$mdxi->Order->BillingAddr->Name     = 'John Doe';
$mdxi->Order->BillingAddr->Street   = 'Foo Street';
$mdxi->Order->BillingAddr->Zip      = '999';
$mdxi->Order->BillingAddr->City     = 'Foo City';
$mdxi->Order->BillingAddr->Country  = 'Foo Country';
$mdxi->Order->BillingAddr->Email    = 'test@yahoo.co,';
$mdxi->Order->BillingAddr->Phone    = '3334444555';
$mdxi->Order->BillingAddr->setMode("ReadOnly");
$mdxi->Order->BillingAddr->Name->setGender("M")->setBirthday("1980-08-08");

$mdxi->Order->URL->Success      = 'https://showcase.mpay24.com/confirm/success.php';
$mdxi->Order->URL->Error        = 'https://showcase.mpay24.com/confirm/error.php';
$mdxi->Order->URL->Confirmation = 'https://showcase.mpay24.com/confirm/confirmation.php';
```
Should generate following XML:
```
<?xml version="1.0" encoding="UTF-8"?>
<Order>
  <Tid>123 TID</Tid>
  <TemplateSet CSSName="MODERN" Language="DE"/>
  <ShoppingCart>
    <Description>Test Payment</Description>
    <Item>
      <Number/>
      <ProductNr>abc</ProductNr>
      <Description>Test Product 1</Description>
      <Quantity>1</Quantity>
      <Price>0.25</Price>
    </Item>
    <Item>
      <Number/>
      <ProductNr>abc</ProductNr>
      <Description>Test Product 1</Description>
      <Quantity>1</Quantity>
      <Price>0.75</Price>
    </Item>
  </ShoppingCart>
  <Price>1.00</Price>
  <Customer Id="10002" UseProfile="true">Max Muster</Customer>
  <BillingAddr Mode="ReadOnly">
    <Name Gender="M" Birthday="1980-08-08">John Doe</Name>
    <Street>Foo Street</Street>
    <Zip>999</Zip>
    <City>Foo City</City>
    <Country>Foo Country</Country>
    <Email>test@yahoo.co,</Email>
    <Phone>3334444555</Phone>
  </BillingAddr>
  <URL>
    <Success>https://showcase.mpay24.com/confirm/success.php</Success>
    <Error>https://showcase.mpay24.com/confirm/error.php</Error>
    <Confirmation>https://showcase.mpay24.com/confirm/confirmation.php</Confirmation>
  </URL>
</Order>
```

**Comment by mdaskalov** [comment]
fixed in v5.0.1
